tests/Keystore/StoredKeyTests.cpp

```diff
@@ -142,19 +144,27 @@ TEST(StoredKey, AddRemoveAccount) {
 
     {
         const auto derivationPath = DerivationPath("m/84'/0'/0'/0/0");
-        key.addAccount("bc1q375sq4kl2nv0mlmup3vm8znn4eqwu7mt6hkwhr", coinTypeBc, derivationPath, "zpub6qbsWdbcKW9sC6shTKK4VEhfWvDCoWpfLnnVfYKHLHt31wKYUwH3aFDz4WLjZvjHZ5W4qVEyk37cRwzTbfrrT1Gnu8SgXawASnkdQ994atn");
+        key.addAccount("bc1qaucw06s3agez8tyyk4zj9kt0q2934e3mcewdpf", coinTypeBc, derivationPath, "zpub6rxtad3SPT1C5GUDjPiKQ5oJN5DBeMbdUR7LrdYt12VbU7TBSpGUkdLvfVYGuj1N5edkDoZ3bu1fdN1HprQYfCBdsSH5CaAAygHGsanwtTe");
```

Comment:
any reason this changed?

Reply:
Old value actually didn't match the test recovery phrase